README.md

```diff
@@ -60,8 +60,8 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 Mladan Ilic Alice Mirigo Boniface Dennis<a name="about-project"></a>
-**Awesome books** In this project, you will build a basic website that allows users to add/remove books from a list. You will achieve that by using JavaScript objects and arrays. You will also need to dynamically modify the DOM and add basic events.
+# 📖 Mladan Ilic <a name="about-project"></a>
```

Comment:
- [ ] KIndly make sure to put here the name of the project not your name. Change your name to the project name in order to keep your project README professional.

Reply:
I would like to tell you that I am grateful for pointing out my weaknesses and that I hope I have corrected all of them according to yours suggestions, except for the part with linters, since it is allowed to use them in the task. TIP: If you separated the code in multiple files and you have issues with the linter, such as complaints about multiple classes in the same file or unused variables, you can disable the linter check for that rule in the file with /* eslint-disable rule-you-want-to-disable */.